components/title/multisize/src/index.js

```diff
@@ -0,0 +1,79 @@
+import React, {PropTypes} from 'react'
+import cx from 'classnames'
+
+/**
+ * Builds an HTML/React Element depending on the size declared (s: H3, m: H2, l: H1).
+ * @param size {string} Size (s, m, l) of title element.
+ * @param className {string} CSS classNames to apply to element.
+ * @param content {element} Children of title element. Could be an element (HTML/React Element), string, or an array of both.
+ * @returns {element} React Title Element.
+ * @private
+ */
+const _buildTitleElement = (size, className, content) => {
+  let element = null
```

Comment:
I had a previous version similar to this, but changed it as an if with the 3 sizes was more explicit. I any case, I can do it like this as well, thanks!